codalab/lib/bundle_util.py

```diff
@@ -354,3 +354,49 @@ def check_bundle_freezable(bundle):
             'Cannot freeze bundle %s(%s), bundle is not in a final state.'
             % (bundle.uuid, bundle.metadata.to_dict()["name"])
         )
+
```

Comment:
Update this test: https://github.com/codalab/codalab-worksheets/blob/master/tests/unit/rest/bundles_test.py#L43

```
data[0]["attributes"].pop("state_details")
```